src/pages/tutorials/build-an-nft-marketplace/part-4.md

```diff
@@ -1457,7 +1460,7 @@ For buying,
 
 To add more collections, go to the Mint page and repeat the process.
 
-## Conclusion
+## Summary
 
 You are able to use any NFT template from the Ligo library.
```

Comment:
Any more info here? Are there other templates in the library that the tutorial should mention?

Reply:
Not sure about this.

Reply:
I just wondered if  there were other templates in the library that we could mention as next steps, like if you wanted to set up DeFi, maybe there's a template for that, or other token types.

Reply:
We use the generic @ligo/fa for both Defi and NFT